lib/omniauth/strategies/cronofy.rb

```diff
@@ -46,6 +46,17 @@ def callback_url
       def raw_info
         @raw_info ||= access_token.get("#{::OmniAuth::Strategies::Cronofy.api_url}/v1/account").parsed['account']
       end
+
+      alias :old_request_phase :request_phase
+
+      def request_phase
+        link_token = session['omniauth.params']['link_token']
+        if link_token && !link_token.empty?
+          options[:authorize_params] ||= {}
+          options[:authorize_params].merge!(:link_token => link_token)
+        end
+        old_request_phase
```

Comment:
Should be able to call `super` here rather than having to do an alias.

Seems strange that this is so messy. We didn't need as much when doing [similar for service accounts](https://github.com/cronofy/omniauth-cronofy/blob/838ec00effa6422047f50aea0d4ac7b2415d3334/lib/omniauth/strategies/cronofy_service_account.rb#L11-L14) but that may not be as robust as this.